bank-info-fe/src/utils/highlightText.js

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
This part is totally optional but serves the purpose of showing user that filtering is happening as the type.